engine/server/webapp/asset-manifest.json

```diff
@@ -1,13 +1,13 @@
 {
   "files": {
     "main.css": "/static/css/main.7b6afa70.css",
-    "main.js": "/static/js/main.caf7863c.js",
+    "main.js": "/static/js/main.2f3b687c.js",
     "index.html": "/index.html",
     "main.7b6afa70.css.map": "/static/css/main.7b6afa70.css.map",
-    "main.caf7863c.js.map": "/static/js/main.caf7863c.js.map"
+    "main.2f3b687c.js.map": "/static/js/main.2f3b687c.js.map"
   },
   "entrypoints": [
     "static/css/main.7b6afa70.css",
-    "static/js/main.caf7863c.js"
+    "static/js/main.2f3b687c.js"
   ]
 }
```

Comment:
nit: missing trailing newline

Reply:
this is auto-generated files; so will ignore that.